include/Z2AudioLib/Z2AudioMgr/Z2AudioMgr.h

```diff
@@ -0,0 +1,15 @@
+#include "global.h"
```

Comment:
```suggestion
#ifndef __Z2_AUDIO_MGR_H_
#define __Z2_AUDIO_MGR_H_
#include "global.h"
```

missing include guard